Makefile

```diff
@@ -0,0 +1,14 @@
+up-bg:
+	docker compose up -d db redis rabbitmq
+
+run-api:
+	docker compose run --rm --service-ports --no-deps api
+
+run-worker:
+	docker compose run --rm --service-ports --no-deps worker
+
+migrate:
+	docker compose run --rm --service-ports --no-deps api aerich upgrade
+
+add-migration:
+	docker compose run --rm --service-ports --no-deps api aerich migrate
```

Comment:
up-bg will start the whole backend?

Reply:
If not, can't you add, for example, `run-backend`?

Reply:
up-bg only runs background services. To run the whole backend just use docker compose up